rust/src/quic/quic.rs

```diff
@@ -35,12 +37,16 @@ use std::collections::VecDeque;
 use std::ffi::CString;
 use suricata_sys::sys::{
     AppLayerParserState, AppProto, SCAppLayerParserConfParserEnabled,
-    SCAppLayerParserRegisterLogger, SCAppLayerProtoDetectConfProtoDetectionEnabled,
+    SCAppLayerParserRegisterLogger, SCAppLayerParserStateSetFlag,
+    SCAppLayerProtoDetectConfProtoDetectionEnabled,
 };
 use tls_parser::TlsExtensionType;
 
 static mut ALPROTO_QUIC: AppProto = ALPROTO_UNKNOWN;
 
+static mut ENCRYPTION_BYPASS_ENABLED: SshEncryptionHandling =
+    SshEncryptionHandling::SSH_HANDLE_ENCRYPTION_TRACK_ONLY;
```

Comment:
TODO: rename and move `SshEncryptionHandling` to something generic in a follow-up commit